src/views/AgmLocationChg.vue

```diff
@@ -416,6 +414,14 @@ export default class AgmLocationChg extends Mixins(CommonMixin, DateMixin,
     return (this.totalFee > 0)
   }
 
+  /** Array of validations rules for AGM location. */
+  get agmLocationRules (): Array<(val) => boolean | string> {
+    const rules = [] as Array<(val) => boolean | string>
+    rules.push(val => !!val || 'AGM location is required.')
+    rules.push(val => (val.length <= 100) || 'Must be less than 100 characters.')
```

Comment:
Spoke to Yui about the rules here:
![yui validation](https://github.com/bcgov/business-filings-ui/assets/122301442/36e95a46-ab31-40d8-8f5e-caf1409a5e08)